AK/CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ swizzle_target_properties_for_swift(simdutf::simdutf)
 target_link_libraries(AK PRIVATE simdutf::simdutf)
 
 # FIXME: Make this generic for all imported shared library dependencies and apply globally
-if (NOT CMAKE_SKIP_INSTALL_RULES AND NOT "${VCPKG_INSTALLED_DIR}" STREQUAL "")
+if (BUILD_SHARED_LIBS AND NOT CMAKE_SKIP_INSTALL_RULES AND NOT "${VCPKG_INSTALLED_DIR}" STREQUAL "")
```

Comment:
In theory the install here is already a no-op for static libs

Reply:
🤷‍♂️ I added this check because we entered this branch on my fork with static artifacts

Reply:
Fair nuff. CMake magic is called magic for a reason